src/status_im/ui/screens/home/animations/responder.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im.ui.screens.home.animations.responder
+  (:require [status-im.ui.components.react :refer [animated pan-responder]]
```

Comment:
please use `:as` instead `:refer`

Reply:
and full names `animation`

Reply:
do you mean using `:as` across the project instead of `:refer`? why ? This `animated` is a [native class](https://facebook.github.io/react-native/docs/animated.html) in react native. But I will remove it, because it is not used in the file 😅.

Reply:
yes, using` :as` across the project instead of `:refer`

Reply:
`animation` comment for next line